uncontrast_studies/management/commands/load_uncon_data.py

```diff
@@ -23,13 +23,16 @@
     NumericListError,
 )
 from contrast_api.data_migration_functionality.helpers import get_list_from_excel
-from contrast_api.data_migration_functionality.studies_parsing_helpers import ProblemInStudyExistingDataException
+from contrast_api.data_migration_functionality.studies_parsing_helpers import (
+    ProblemInStudyExistingDataException,
+    MissingCountryDetectionException,
+)
 from uncontrast_studies.services.process_uncon_row import process_uncon_row
 from uncontrast_studies.services.errors_logger import write_errors_to_log
 
 logger = logging.getLogger("UnConTrast")
```

Comment:
generally it's a better practice "usually" (unless you have a good reason) to use this format:
```python
logging.getLogger(__name__)
```
Which automatically makes the logger name the full file path, and helps pinpoint the exact place of the log